### DISCUSSION TITLE
AUC calculation

### DISCUSSION THREAD

**Discussion by Pagdeklaver** [LAPKB/Pmetrics#216] [Q&A]
Hi, 
im wondering how I can calculate an AUC from a dataset consisting of id-time-concentration

**Comment by VishnuEriyat** [comment]
Hi there, I'm also working on similar lines using some vancomycin data. Have you tried passing on the data csv file to "makeNCA(data)". If you make it into an object like x<- makeNCA(data) and then use write.csv(x,"drive name:/folder name/filename.csv"), it will give a NCA output.


Actually I tried this but faced another issue, the AUC thus predicted and the AUC calculated manually by trapezoid method are different. Why does it happen Im trying to find out.

**Comment by mnneely** [comment]
We have validated makeAUC against other packages that calculate it. Can you post more details with an example from one curve how your calculations differ?

**Reply by VishnuEriyat** [reply]
Hi, sorry for the late reply, herewith attached is the snipshots from
1. model file, 2. the makeNCA output, 3.the way I calculated AUC manually
![image](https://github.com/LAPKB/Pmetrics/assets/153790648/60df3b2e-a80e-4170-bda3-1f92f132415e)
![image](https://github.com/LAPKB/Pmetrics/assets/153790648/0bdfc99b-9aa4-437d-94ac-d9831cc5d05e)
![image](https://github.com/LAPKB/Pmetrics/assets/153790648/48c430fd-1844-4738-b17f-adc35298c99e)

**Comment by mnneely** [comment]
If you could post the excerpt from the data as a text file or paste the times and concentrations, that would allow us to copy them.  Thanks.

**Reply by VishnuEriyat** [reply]
Will this suffice

[vanco data.txt](https://github.com/LAPKB/Pmetrics/files/15004899/vanco.data.txt)
